docs/local-development.md

```diff
@@ -28,4 +28,6 @@
     fin drupal-install
     ```
 
+    If this stage fails due to `RuntimeException: Missing $settings['hash_salt'] in settings.php` - make sure you have the `.env` file in your root folder and there's a random string for the ENV variable `DRUPAL_SALT`
```

Comment:
@ajtrichards Can this be moved into a separate PR?

Reply:
Yeah no problem 👍